zipline/src/jvmMain/resources/META-INF/proguard/zipline.pro

```diff
@@ -0,0 +1,36 @@
+### ZIPLINE
```

Comment:
This file needs set as `consumerProguardFiles` for the Android build to also use them.

Reply:
Oooh fixed. I used Moshi as a sample but it’s a JAR and not an AAR. Should have tried something else.

Reply:
should just be something like `consumerProguardFiles files("src/jvmMain/resources/META-INF/proguard/zipline.pro")` inside... some block.

Reply:
`android.defaultConfig`: https://github.com/JakeWharton/butterknife/blob/fcdebedf3276096db2f51bf6372b849b5a9c75ed/butterknife/build.gradle#L3-L9